framework/include/wifi_manager/wifi_manager.h

```diff
@@ -91,7 +91,7 @@ typedef struct {
 typedef struct {
 	char ssid[32];
 	uint16_t channel;
-	char passphrase[32];
+	char passphrase[64];
```

Comment:
Hi, To avoid this hard coded 32 and 64 and also to avoid the confusion of using 31 and 63, I suggest following approach. Kindly think of it
-----------
#define MAX_SSID 31
#define MAX_PASSPHRASE 63

typedef struct {
 	char ssid[MAX_SSID + 1];   /* this addition 1 will be used for NULL char '\0' */
 	uint16_t channel;
        char passphrase[MAX_PASSPHRASE + 1];  /* same as above */
}

Using MAX_SSID and MAX_PASSPHRASE across all files would increase readability and avoid confusions

Thanks,
Pradeep

Reply:
I will do refactoring when other wifi vendors join Tizen RT.
I hope soon.
Thanks for the recommendation.